src/client/apps/articles_list/client.tsx

```diff
@@ -14,7 +14,9 @@ initWebsocket(store, sd.APP_URL)
 export const init = () =>
   render(
     <Provider store={store}>
-      <ArticlesList articles={sd.ARTICLES} published={sd.HAS_PUBLISHED} />
+      <Theme>
+        <ArticlesList articles={sd.ARTICLES} published={sd.HAS_PUBLISHED} />
```

Comment:
What does wrapping this in `<Theme>` do?

Reply:
This wraps the app in Palette's theme, so it will receive the proper sizes for `m={1}` etc. and breakpoints.

Reply:
👍